### ISSUE TITLE
Auto generated _Sidebar

### ISSUE THREAD

**Issue by ziemsky** [ziemsky/readme2wiki#1]
**TL;DR**: add support for automatic generation of `_Sidebar.md`

Option 1:
`_Sidebar.md` could be generated as a simple, indented list of Markdown files found in Wiki directory, where indentations would corresponding to the levels of Markdown files nesting within the directory structure. In this variant, the plugin would simply search for all Markdown files and build the `_Sidebar.md` fully automatically.

Option 2:
Have a Table of Content section in the main Markdown file and have the plugin interpret it and build `_Sidebar.md` out of it.

**Motivation**
Why have `_Sidebar.md` at all:
1. Having a navigable table of content is useful in a multi-document wiki but the problem of defining such TOC within the main readme/home file itself is that, when clicking such links in Github, the target Markdown documents are rendered as plain text files, outside of Wiki (i.e. in separate tabs). Such links defined in `_Sidebard.md` work correctly.
1. The Pages section lists _all_ Markdown files - top-level ones as well as the ones nested in sub-directories - as a flat list. `_Sidebar.md` allows to visualise a logical structure of individual documents.

Why generate it automatically, when it can be authored manually and stored in the main repo alongside the home/main readme:
1. Because having it generated automatically would mean one less thing to do manually and automation of wiki creation out of Markdown files is the main goal of this plugin.

**Comment by ziemsky** [comment]
Flagging as low priority because, given that the plugin is intended to generate simple Wikis, manual authoring of `_Sidebar.md` should suffice in most cases.
